src/GatewayAPI/Database/TransactionQuerier.cs

```diff
@@ -129,21 +131,31 @@ public record RecentTransactionPageRequest(
     int PageSize
 );
 
+public record ResourceHolder(
```

Comment:
Given this holds more than just Resources, perhaps this naming isn't perfect?

Maybe `JoinedStaticEntityLookup` or something might be a little clearer?

Reply:
Naming is poor, agree. Fun fact: this type was initially named `EntityFrameworkSucks` (..`Hard`) hence `efs` shortcuts used here and there. :D I'll try to come up with something better tomorrow :)